Services/WatchlistService.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  WatchlistService.swift
+//  Movies
+//
+//  Created by Diana Chizhik on 21.07.22.
+//
+
+import UIKit
+
+protocol WatchlistServiceProtocol {
+    func getStatus(for id: Int)-> WatchlistStatus
+    func toggleStatus(for item: WatchlistItem) -> WatchlistStatus
+    func getWatchlistItems() -> [WatchlistItem]?
+}
+
+enum WatchlistStatus {
+    case added, notAdded
+}
+
+class WatchlistService {
+    typealias Watchlist = [Int : WatchlistItem]
+    static var watchlistKey: String = "watchlist"
```

Comment:
A static `var` still allows for modification, in this case we want to avoid that so let's make it `static let`. To understand how this behaves:
```swift
struct Vehicle {
    static var subtype = "car"
    
    let name: String
    
    func printSubtype() {
        print(Self.subtype)
    }
}

let a = Vehicle(name: "Yaris")
let b = Vehicle(name: "Corsa")
a.printSubtype() // prints: "car"
b.printSubtype() // prints: "car"
print(Vehicle.subtype) // prints: "car"

Vehicle.subtype = "not a car"
a.printSubtype() // prints: "not a car"
b.printSubtype() // prints: "not a car"
print(Vehicle.subtype) // prints: "not a car"
```